packages/injectable/src/dependency-injection-container/createContainer.js

```diff
@@ -376,7 +376,7 @@ const getInstance = ({
     ...(isUndefined(instantiationParameter) ? [] : [instantiationParameter]),
   );
 
-  if (instanceKey !== nonStoredInstanceKey) {
+  if (instanceKey !== transientLifecycle) {
```

Comment:
This change of naming hurts self-documentation a bit. Before the naming explained motivation, after it does not.

Reply:
Also this is in breach of OCP.